src/Razor/src/Microsoft.VisualStudio.Editor.Razor/BackgroundDocumentGenerator.cs

```diff
@@ -173,7 +173,6 @@ public void Enqueue(ProjectSnapshot project, DocumentSnapshot document)
                 if (_projectManager.IsDocumentOpen(document.FilePath))
                 {
                     _infoProvider.SuppressDocument(project.FilePath, document.FilePath);
-                    return;
                 }
```

Comment:
Instead of exiting after suppression, we proceed to still generate the doc here.